src/thing/remove.test.ts

```diff
@@ -2290,15 +2603,38 @@ describe("removeLiteral", () => {
     });
   });
 
+  it("accepts time as Literal", () => {
+    const thingWithTime = getMockThingWithLiteralFor(
+      "https://some.vocab/predicate",
+      "13:37:42Z",
+      "time"
+    );
+
+    const updatedThing = removeLiteral(
+      thingWithTime,
+      "https://some.vocab/predicate",
+      DataFactory.literal(
+        "13:37:42Z",
+        DataFactory.namedNode("http://www.w3.org/2001/XMLSchema#time")
+      )
+    );
+
+    expect(
+      updatedThing.predicates["https://some.vocab/predicate"].literals
+    ).toStrictEqual({
+      "http://www.w3.org/2001/XMLSchema#time": [],
+    });
+  });
+
   it("accepts datetime as Literal", () => {
-    const thingWithDatetime = getMockThingWithLiteralFor(
+    const thingWithDateTime = getMockThingWithLiteralFor(
       "https://some.vocab/predicate",
       "1990-11-12T13:37:42Z",
       "dateTime"
     );
 
     const updatedThing = removeLiteral(
-      thingWithDatetime,
+      thingWithDateTime,
```

Comment:
These changes seem unintended? We're spelling it `Datetime` everywhere else if I recall correctly, so for consistency's sake probably better to revert this.

Reply:
yep, that was unintended...